examples/component-example.py

```diff
@@ -0,0 +1,90 @@
+import pygfx as gfx
+from wgpu.gui.auto import call_later, run, WgpuCanvas
+
+from collagraph import Collagraph, Component, create_element as h, EventLoopType
+from collagraph.renderers import PygfxRenderer
+
+
+class Button(Component):
+    geometry = gfx.box_geometry()
+    materials = {
+        "default": gfx.MeshPhongMaterial(color=[1.0, 0.5, 0.0]),
+        "pressed": gfx.MeshPhongMaterial(color=[0.0, 0.5, 0.0]),
+    }
+
+    def __init__(self, props):
+        super().__init__(props)
+        self.state["pressed"] = False
+        self.state["scale"] = self.props.get("scale", [0.85] * 3)
+        self.mesh = None
```

Comment:
Just wanted to say that this looks great, it's very clear now that `self.mesh` is not a prop, not state, and not reactive.

Reply:
Yes! I also believe that it is unused currently, so I'll remove it anyways ;)